Dockerfile.bridge

```diff
@@ -0,0 +1,22 @@
+ARG ALPINE_IMAGE=alpine
+ARG ALPINE_VERSION=3.15
+ARG ZT_COMMIT=eac56a2e25bbd27f77505cbd0c21b86abdfbd36b
+ARG ZT_VERSION=1.8.4
+
+FROM zyclonite/zerotier
+
+RUN apk add --no-cache --purge --clean-protected --update supervisor iptables \
+  && mkdir -p /var/log/supervisor \
+  && rm -rf /var/cache/apk/*
+
+ENV LOG_PATH=/var/log/supervisor \
+    BRIDGE=false
```

Comment:
as this will be anyway a different image tag, the environment variable can be removed as well